### PR TITLE
Adding rules to maintain CHANGELOG of Helm chart

### PR DESCRIPTION
**Pull request by gantrior**

### CODE REVIEW

**Review by etichy — APPROVED** [review]

File: doc/development.md [hunk 1]
```diff
@@ -97,13 +106,21 @@ Steps to publish new version:
...
+   1. Create release record with the right version and the date.
+   2. Write all changes recorded in `Unreleased` section into the release.
+4. Create PR for the changes to the `master` branch and merge them.
+5. Run "Release Helm Chart" GitHub action workflow.
+6. Find relevant release in GitHub, edit it and write all changes recorded into [CHANGELOG.md](deploy/helm/CHANGELOG.md) into its description.
+7. Review PR that was created for the changes to the `gh-pages` branch and merge them.
+
+   ```shell
+   helm template swo-k8s-collector deploy/helm -n="<NAMESPACE>" --set-string externalRenderer=true > deploy/k8s/manifest.yaml
+   ```
```
Comment (pstranak-sw) [inline doc/development.md:126]: Why is this here?
